src/components/policy/policy_external/src/policy_manager_impl.cc

```diff
@@ -574,6 +570,10 @@ CheckAppPolicyResults PolicyManagerImpl::CheckPermissionsChanges(
     const std::shared_ptr<policy_table::Table> snapshot) {
   LOG4CXX_AUTO_TRACE(logger_);
 
+  // Replace predefined policies with its actual setting, e.g. "123":"default"
+  // to actual values of default section
+  UnwrapAppPolicies(pt_update->policy_table.app_policies_section.apps);
```

Comment:
UnwrapAppPolicies is moved in the external policies, but below in regular the method is removed?

Reply:
@JackLivio `UnwrapAppPolicies` is already in `CheckPermissionsChanges` for policy_regular. So the removed one is redundant.